test/subscriber_test.rb

```diff
@@ -83,7 +92,7 @@ class SubscriberTest < Test::Unit::TestCase
         { :EmailAddress => "example+2@example.com", :Name => "Example Two" },
         { :EmailAddress => "example+3@example.com", :Name => "Example Three" },
       ]
-      import_result = CreateSend::Subscriber.import @auth, @list_id, subscribers, true, true
+      import_result = CreateSend::Subscriber.import @auth, @list_id, subscribers, true, "No", true
```

Comment:
As per https://www.campaignmonitor.com/api/subscribers/#importing-many-subscribers, "ConsentToTrack" field is coming with the subscriber record and not passing into the import method. Should we add ConsentToTrack into subscribers array and remove the ConsentToTrack boolean value here?